_posts/patterns/2015-12-11-tables.md

```diff
@@ -0,0 +1,257 @@
+          <th>Strengths</th>
+          <th>Primary Inhabitants</th>
+          <th>No. Inhabitants</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <th>Pernrith</th>
+          <td>Ancient water source</td>
+          <td>Low economy</td>
+          <td>Rare animal training and rare herbalism</td>
+          <td>People</td>
+          <td class="align-right">15</td>
+        </tr>
+        <tr>
+          <th>Fandralore</th>
+          <td>Strong metal ores</td>
+          <td>Armorsmithing</td>
+          <td>Sustainable hunting and prosperous trade</td>
+          <td>Elves</td>
+          <td class="align-right">20</td>
+        </tr>
+        <tr class="highlight">
+          <th>Kasmord</th>
+          <td>Precious gems</td>
+          <td>Medicine</td>
+          <td>Strong magicians and refined baking</td>
+          <td>Elves</td>
+          <td class="align-right">25</td>
+        </tr>
+      </tbody>
+      <tfoot>
+        <tr class="align-right">
+          <th colspan="5">Total World Inhabitants</th>
+          <td class="align-right">60</td>
+        </tr>
+      </tfoot>
+    </table>
+  </div>
+</div>
+
+
+<h3 class="hd-6 example-set-hd">Responsive (with Column hiding)</h3>
+<div class="example-set">
+  <table class="table table-responsive">
+    <colgroup class="colgroup">
+      <col class="col col1">
+    </colgroup>
+    <colgroup class="colgroup">
+      <col class="col col2">
+      <col class="col col3">
+      <col class="col col4">
+      <col class="col col5">
+    </colgroup>
+    <colgroup class="colgroup">
+      <col class="col col6">
+    </colgroup>
+    <thead>
+      <tr>
+        <th>Town Name</th>
+        <th>Founding feature</th>
+        <th class="supplemental">Weaknesses</th>
+        <th class="supplemental">Strengths</th>
+        <th>Primary Inhabitants</th>
+        <th>No. Inhabitants</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <th>Pernrith</th>
+        <td>Ancient water source</td>
+        <td class="supplemental">Low economy</td>
+        <td class="supplemental">Rare animal training and rare herbalism</td>
+        <td>People</td>
+        <td>15</td>
+      </tr>
+      <tr>
+        <th>Fandralore</th>
+        <td>Strong metal ores</td>
+        <td class="supplemental">Armorsmithing</td>
+        <td class="supplemental">Sustainable hunting and prosperous trade</td>
+        <td>Elves</td>
+        <td>20</td>
+      </tr>
+      <tr>
+        <th>Kasmord</th>
+        <td>Precious gems</td>
+        <td class="supplemental">Medicine</td>
+        <td class="supplemental">Strong magicians and refined baking</td>
+        <td>Elves</td>
+        <td>25</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+
+<h3 class="hd-6 example-set-hd">Sortable</h3>
+<div class="example-set">
+  <table class="table table-sortable">
+    <tr>
+      <th class="sortable sort-current sort-descending" aria-label="Sorted Descending">
```

Comment:
@frrrances ```<th>``` elements don't have an accessible name so the aria-label won't be interpreted here (but it's a good idea!).  Let's try to inform the user some other way, like using offscreen text.  Perhaps we can toggle the offscreen text value to indicate what pressing the ```<button>``` will do e.g. ```Sort<span class=sr>&nbsp;Ascending</span>```  and then toggle that value to Descending on press

cc @talbs